frontend/src/bundles/talent-onboarding/components/candidate-profile/profile-second-section/profile-second-section.tsx

```diff
@@ -0,0 +1,178 @@
+import {
+    Button,
+    FormControl,
+    Grid,
+    RadioGroup,
+    Typography,
+} from '~/bundles/common/components/components.js';
+import { getValidClassNames } from '~/bundles/common/helpers/helpers.js';
+import { useAppForm } from '~/bundles/common/hooks/hooks.js';
+import { CandidateParameter } from '~/bundles/talent-onboarding/components/components.js';
+import { CandidateIcons } from '~/bundles/talent-onboarding/enums/enums.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+    // replace with real data type
```

Comment:
```suggestion
    // TODO: replace with real data type
```

Reply:
if you add `TODO` it is much easier to find it after

Reply:
done